### DISCUSSION TITLE
Using WireUI and Livewire to open a modal without magic $openModal('blurModal') method

### DISCUSSION THREAD

**Discussion by JohnMSykes** [wireui/wireui#441] [Q&A]
Hi - I am relatively new to the Laravel, Livewire, Alpine universe. I was able to take baby steps to get a working example of a blurModal going after much n00b thrashing. In the end, I used the magic `$openModel('blurModal')` function. No x-data, no changing` show = ! show`, etc. It felt like cheating and remains magical, because I don't fully understand how to leverage Livewire to set show to true, etc.
Also, there was a Livewire error in the browser console, and I don't know if that was indicative of a deeper misunderstanding on my part. To set this up, I created a new Laravel app and ran via `php artisan serve`. I used `php artisan make:livewire BlurModal` to create the http and resource components. I am using Vite with `npm run dev` to support rapid development.

**Page containing @livewire directives:**
![welcome blade php](https://user-images.githubusercontent.com/14906331/185467629-134f9e07-29d9-4080-9e2a-c41625100b06.png)

**Browser Console Warning**
![browser-console-warning](https://user-images.githubusercontent.com/14906331/185467859-7f8ba87c-18cf-40f4-9279-5617618c1cc6.png)

**BlurModal.php Component Class File**
![BlurModal php](https://user-images.githubusercontent.com/14906331/185467989-56cef266-5ea4-4221-be06-fd925d2e5e49.png)

**blur-modal.blade.php Blade View File**
![blur-modal blade php](https://user-images.githubusercontent.com/14906331/185468121-0b5fed5c-0cb8-457c-a8f1-6a2f617d02ec.png)

Thanks in advance for your thoughts or guidance on a more elegant and correct way to use WireUI in a more artful and sophisticated way. It is a great body of work and we're hoping to use it in a much more sophisticated way.

**Comment by JohnMSykes** [comment]
I should have included this image of the modal working (Yay!) - It *DOES* work with my facile approach to WireUI. A win is a win, eh?! :-)
![modal-display-in-browser](https://user-images.githubusercontent.com/14906331/185469015-d628dd32-5686-4dfd-85d7-04f4b4728bb7.png)

**Comment by ph7jack** [comment]
@JohnMSykes Look at the livewire properties https://laravel-livewire.com/docs/2.x/properties

The modal can be used with the livewire properties, using a wire:model directive
